cinema/serializers.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=55)
```

Comment:
The `title` field in the serializer has a `max_length` of 55, which is inconsistent with the `max_length` of 255 in the `Movie` model. Consider updating the serializer to match the model's `max_length`.